unittests.py

```diff
@@ -28,21 +24,22 @@ def tealiumCallback(self, info, success, error=None):
                                   "tealium_event_type"]
 
         sentURL = info['encoded-url']
+        print(sentURL)
```

Comment:
Leftover print statement?  Either prefix it with something like "Tealium Python 1.2.0: sent dispatch to url: "  or can remove.